.storybook/stories/forms/forms-reactive.stories.ts

```diff
@@ -26,13 +26,11 @@ export default {
   ],
   argTypes: {
     // inputs
-    clrLabelSize: { control: { type: 'number', min: 1, max: 12 } },
+    clrLabelSize: { type: 'number', min: 1, max: 12 },
```

Comment:
```suggestion
    clrLabelSize: { control: { type: 'number', min: 1, max: 12 } },
```

This is wrong. Please revert.